client/me/purchases/manage-purchase/purchase-meta.jsx

```diff
@@ -455,7 +455,7 @@ function PurchaseMetaExpiration( {
 	const translate = useTranslate();
 	const moment = useLocalizedMoment();
 	const isProductOwner = purchase?.userId === useSelector( getCurrentUserId );
-	const isAutorenewalEnabled = purchase ? ! isExpiring( purchase ) : null;
+	const isAutorenewalEnabled = purchase?.isAutoRenewEnabled;
```

Comment:
Maybe default this to `false` to ensure we have a boolean value?
```suggestion
	const isAutorenewalEnabled = purchase?.isAutoRenewEnabled ?? false;
```